packages/workspace/src/browser/workspace-commands.ts

```diff
@@ -447,6 +447,14 @@ export class WorkspaceRootUriAwareCommandHandler extends UriAwareCommandHandler<
         super(selectionService, handler);
     }
 
+    isEnabled(): boolean {
```

Comment:
We should not override it. Why do we need it for this PR?

Reply:
I've updated for the moment in order to expose commands to the root of the workspace mimicking if a true root node was present. The `WorkspaceRootUriAwareCommandHandler` determines if a root node is selected but unfortunately this is impossible now with the newest implementation of the explorer.